### PR TITLE
Feature/activity class

### PR DESCRIPTION
**Pull request by K-Howard**

### CODE REVIEW

**Comment by K-Howard** [comment]
#### What's this PR do?
Adds activity methods and more tests as well as more data to test.
#### Where should the reviewer start?
Activity.js and activity-test
#### How should this be manually tested?
npm test
#### Any background context you want to provide?
Need another method or 2

#### Questions:
other things to test?

File: README.md [hunk 1]
```diff
@@ -1,78 +1,61 @@
...
-    "hoursSlept": [number - hours],
-    "sleepQuality": [number - unitless]
-  },
-  ...more sleep data
-]
-```
+# FitLit
+
+# Amelia, Corinne, Kyle, and Megan
+
```
Comment (ameliaeiger) [inline README.md:4]: Needs a label for setup instructions

File: test/Activity-test.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+   ])
+  });
+
+  it("should have a method that finds step climbing record", function () {
+    expect(activity.stairClimbRecord(1)).to.equal(36)
+  });
+
+//   it.skip('should do this' () => {
+//     expect().to.();
+//   });
```
Comment (megschuetz) [inline test/Activity-test.js:53]: Could leave any notes like out. Other than that looks good.

File: README.md [hunk 3]
```diff
@@ -1,78 +1,61 @@
...
+- Mocha
+- Chai
+- Webpack
+- Fetch API
+- WAI-ARIA
+
+#### Instructions on Use
+
+
+### Demo of LitFit
```
Comment (ameliaeiger) [inline README.md:48]: FitLit

File: src/user.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+class User {
```
Comment (ameliaeiger) [inline src/user.js:1]: I believe this is our file we were not using for user and which Megan had already deleted. Let's avoid re-adding this file to the repo.

**Review by ameliaeiger — CHANGES_REQUESTED** [review]
Let's avoid uploading the user.js file that Megan has deleted. Our working user.js file is User.js

File: src/Activity.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        return user;
+      }
+    }).minutesActive;
+    return minsActivity;
+  };
+
+  activityStepsForWeek(userId) {
+    const userData = this.findUser(userId);
+    const lastElement = userData.indexOf(userData[userData.length - 1]);
+    // const userData = this.findUser(userId);
```
Comment (megschuetz) [inline src/Activity.js:28]: Probably just remove line 28 here.

**Review by ameliaeiger — APPROVED** [review]

**Pull request merged**